workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/BuildConfigEditorStyle.cs

```diff
@@ -0,0 +1,115 @@
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.BuildSystem.Configuration
+{
+    internal class BuildConfigEditorStyle
+    {
+        internal readonly GUIContent
+            AddWorkerTypeButtonContents = new GUIContent(string.Empty, "Add worker type");
+
+        internal readonly GUIContent RemoveWorkerTypeButtonContents =
+            new GUIContent(string.Empty, "Remove worker type");
+
+        internal readonly GUIContent AddSceneButtonContents = new GUIContent(string.Empty, "Add scene");
```

Comment:
super nit: the convention is to always capitalize Unity terms: -> "Add Scene" and "Remove Scene"